crypto/fipsmodule/cpucap/cpu_aarch64_linux.c

```diff
@@ -133,19 +131,14 @@ void OPENSSL_cpuid_setup(void) {
   // Before calling armv8_cpuid_probe and reading from MIDR_EL1 check that it
   // is supported. As of Valgrind 3.21 trying to read from that register will
   // cause Valgrind to crash.
-
-#if defined(OPENSSL_ANDROID)
   if (hwcap & kCPUID) {
```

Comment:
Why does `kCPUID` even need to be here? It's a `static const` value initialized above but otherwise doesn't appear to be used?

Reply:
I'm not following, it's used here `f (hwcap & kCPUID) {`? This is how we have it structured on main, so if you think the use of constants should be cleaned up we should take it up there first.

Reply:
it's not a boolean above. it's the bitwise and `&` :)

Reply:
🤦‍♂️